java-checks/src/main/java/org/sonar/java/checks/tests/AssertJConsecutiveAssertionCheck.java

```diff
@@ -57,7 +57,7 @@
 
   public static final MethodMatchers ASSERTJ_SET_CONTEXT_METHODS = MethodMatchers.create()
     .ofSubTypes("org.assertj.core.api.AbstractAssert")
-    .name(name -> name.startsWith("extracting") || name.startsWith("using") || name.startsWith("filtered"))
+    .name(name -> name.startsWith("extracting") || name.startsWith("using") || name.startsWith("filtered") || "flatExtracting".equals(name))
```

Comment:
Unless there is a reason to believe that `name` might be null, which should be tested before entering the predicate, it might be better to change the line to `name.equals("flatExtracting")` for stylistic consistency.